src/crypto/litecoin.ts

```diff
@@ -40,23 +42,38 @@ class Litecoin implements ICrypto {
     return String(address);
   }
 
-  getAddress(derivationMode: string, xpub: string, account: number, index: number): string {
+  getNativeSegWitAddress(xpub: string, account: number, index: number): string {
+    const { address } = bjs.payments.p2wpkh({
+      pubkey: bip32.fromBase58(xpub, this.network).derive(account).derive(index).publicKey,
+      network: this.network,
+    });
+
+    return String(address);
+  }
```

Comment:
@hzheng-ledger I just copy/pasted this from Bitcoin, can you please confirm this is correct for Litecoin Native SegWit ?